irohad/ametsuchi/impl/tx_cache_outcome.cpp

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ametsuchi/tx_cache_outcome.hpp"
+
+using namespace iroha::ametsuchi;
+
+const TxCacheOutcome::HashType &TxCacheOutcome::HashContainer::hash() const {
+  return hash_;
+}
+
+const TxCacheOutcome::OutcomeType &TxCacheOutcome::outcome() const {
```

Comment:
```suggestion
const TxCacheOutcome::TxCacheStatus &TxCacheOutcome::outcome() const {
```
details are below (comment to the line 63 in `tx_cache_outcome.hpp`)

Reply:
```suggestion
const TxCacheResponse::OutcomeType &TxCacheResponse::outcome() const {
```
please see details in the comment on the line 22 in `tx_cache_outcome.hpp`

Reply:
The idea is the same (see comment on the line 22 in `tx_cache_outcome.hpp`)

```suggestion
const TxCacheOutcome::OutcomeType &TxCacheOutcome::status() const {
```